libraries/aws_iam_root_user.rb

```diff
@@ -45,6 +80,22 @@ def exists?
     !@summary_account.empty?
   end
 
+  def has_account_signing_certificate?
+    @summary_account['AccountSigningCertificatesPresent'].present?
+  end
+
+  def has_MFA_devices_in_use?
+    @summary_account['MFADevicesInUse'] >= 1
+  end
+
+  def has_policy_versions_in_use?
+    @summary_account['PolicyVersionsInUse'] >= 1
+  end
+
+  def has_min_users?
+    @summary_account['Users'] >= 1
+  end
```

Comment:
What is this? Return true if there are MORE than one user?  That is backwards.

Reply:
Removed the matcher